packages/esm-patient-registration-app/src/patient-registration/validation/patient-registration-validation.tsx

```diff
@@ -34,7 +34,16 @@ export function getValidationSchema(config: RegistrationConfig) {
       otherwise: Yup.number().nullable(),
     }),
     monthsEstimated: Yup.number().min(0, 'negativeMonths'),
-    deathDate: Yup.date().max(Date(), 'deathdayNotInTheFuture').nullable(),
+    deathDate: Yup.date()
+      .nullable()
+      .max(new Date(), 'deathdayNotInTheFuture')
+      .test('deathDate-after-birthdate', 'deathdayInvalidDate', function (value) {
+        const { birthdate } = this.parent;
+        if (value && birthdate) {
+          return value >= birthdate;
+        }
+        return true;
+      }),
```

Comment:
Great, exactly what was needed 👏 

You can also remove the `.nullable()` check on the basis of the configuration's `validation.required`